client/src/components/profile-card.js

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from 'react';
+import { inject } from '../utils';
+import { Card, Icon } from 'antd';
+import './profile-card.scss';
+import { AuthStore } from '../stores';
+
+function sleep(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+const ProfileCard = ({ stores: [authStore], ...props }) => {
+  let [loading, setLoading] = useState(false);
+
+  let setBaseInfo = async () => {
```

Comment:
Remove this thing

Reply:
Also, you shouldn't fetch user information from `AuthStore`, since this page is supposed to work with all authors

Reply:
Instead of using native DOM API, you should do things the React way.

e.g,

```js
function Component() {
  let user = //... idk somewhere
  return (
    <div>{user ? user.displayName : 'Placeholder or null'}</div>
    <div>{user ? user.bio : 'Placeholder or null'}</div>
  )
}
```

Reply:
I have made the changes, please take a look. Thank you.